{"1.0":{"info":{"author":"Plone Security Team","author_email":"security@plone.org","bugtrack_url":null,"classifiers":["Framework :: Plone","Programming Language :: Python"],"description_content_type":null,"docs_url":null,"download_url":"UNKNOWN","downloads":{"last_day":-1,"last_month":-1,"last_week":-1},"dynamic":null,"home_page":"https://github.com/plone","keywords":"plone security hotfix patch","license":"GPL","maintainer":null,"maintainer_email":null,"name":"Products.PloneHotfix20151208","package_url":"https://pypi.org/project/Products.PloneHotfix20151208/","platform":"UNKNOWN","project_url":"https://pypi.org/project/Products.PloneHotfix20151208/","project_urls":{"Download":"UNKNOWN","Homepage":"https://github.com/plone"},"provides_extra":null,"release_url":"https://pypi.org/project/Products.PloneHotfix20151208/1.0/","requires_dist":null,"requires_python":null,"summary":"Various Plone hotfixes, 2015-12-08","version":"1.0","yanked":false,"yanked_reason":null},"last_serial":21064546,"urls":[{"comment_text":"","digests":{"blake2b_256":"0c2435e821d3165df79506bec21425f82515c35095afdd6958ba9c0282fd3f5e","md5":"4290c130762831f6d6ca5721b22349ed","sha256":"a92d23481981c70d6af3d4eb0e87db4b09eda391921c0cf516499a38cc2fa598"},"downloads":-1,"filename":"Products.PloneHotfix20151208-1.0.zip","has_sig":false,"md5_digest":"4290c130762831f6d6ca5721b22349ed","packagetype":"sdist","python_version":"source","requires_python":null,"size":16889,"upload_time":"2015-12-08T15:08:55","upload_time_iso_8601":"2015-12-08T15:08:55.501203Z","url":"https://files.pythonhosted.org/packages/0c/24/35e821d3165df79506bec21425f82515c35095afdd6958ba9c0282fd3f5e/Products.PloneHotfix20151208-1.0.zip","yanked":false,"yanked_reason":null}],"vulnerabilities":[]}}